src/cio-config.ts

```diff
@@ -30,3 +30,16 @@ export type CioConfig = {
     siteId: string;
  }
 };
+
+export type CioPushPermissionOptions = {
+  ios?: {
+    badge: boolean;
+    sound: boolean;
+  };
+}
+
+export enum CioPushPermissionStatus {
+  Granted = 'Granted',
+  Denied = 'Denied',
+  NotDetermined = 'NotDetermined',
+}
```

Comment:
```suggestion
export enum CioPushPermissionStatus {
  Granted = 'GRANTED',
  Denied = 'DENIED',
  NotDetermined = 'NOTDETERMINED',
}
```

Not sure why we have this as captalized? in [TS naming convention is uppercassed.](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums)

Reply:
Although uppercasing is a common convention, it’s not a requirement in TypeScript. I chose not to follow it to maintain consistency across platforms and to ensure better clarity and readability in the code.​ 
 
Let me know if you have a strong reason to change it to uppercase ?

Reply:
My preference for uppercase if for two reasons,
- Its common convention 
- Its fail proof, quoting what you said,

> maintain consistency across platforms

There is no certainty, both android and iOS would return `capitalized` cases, so its safer if we just upper case it, so no matter what other platforms are returning our comparison won't fail.

Reply:
Gotcha. It makes sense ! Updated.